tests/acceptance/db_test.go

```diff
@@ -1,793 +1,15 @@
 package tests
```

Comment:
A lot of this code is no longer used.  It exists only because it is used in the test scripts.  Removing the tests as it is deprecated and adds to the issues to the testing system.